src/Files.App/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -169,9 +169,9 @@ public async Task UpdateLabelAsync()
 		{
 			try
 			{
-				var properties = await Root.Properties.RetrievePropertiesAsync(new[] { "System.ItemNameDisplay" })
-					.AsTask().WithTimeoutAsync(TimeSpan.FromSeconds(5));
-				Text = (string)properties["System.ItemNameDisplay"];
+				var properties = Root is not null ? await Root.Properties.RetrievePropertiesAsync(new[] { "System.ItemNameDisplay" })
+                    .AsTask().WithTimeoutAsync(TimeSpan.FromSeconds(5)) : null;
+                Text = (string?)properties?["System.ItemNameDisplay"];
```

Comment:
Looks like these lines might be using spaces? other than that it LGTM.

Reply:
Yes, half of the issues with this pr are my vs formatter dying randomly